internal/snap_api/snap_api_test.go

```diff
@@ -145,8 +146,7 @@ old = Promise;
 				ProjectBaseDir + `/entry.js`: `
 __commonJS["./entry.js"] = function(exports, module2, __filename, __dirname, require) {
   "use strict";
-let __get_old__;
-  var old;
+let __get_old__;  var old;
   
```

Comment:
```suggestion

```

Reply:
We talked about this offline, but basically in order to fix sourcemaps for now, I had to mess with the formatting of the generated code. The generated code is functional and properly sourcemapped but just doesn't look 100% formatted correctly. I logged this to address this later: https://github.com/cypress-io/esbuild/issues/11

Reply:
Added comment for this here: [`f972bb9` (#10)](https://github.com/cypress-io/esbuild/pull/10/commits/f972bb944f1be3e859325b53b7fee7b275ec5097)